src/services/item/plugins/tag/controller.test.ts

```diff
@@ -203,6 +207,67 @@ describe('Tag Endpoints', () => {
           payload: { name: 'name', category: TagCategory.Discipline },
         });
 
+        expect(response.statusCode).toBe(StatusCodes.FORBIDDEN);
+      });
+    });
+  });
+  describe('DELETE /:itemId/tags/:tagId', () => {
+    it('Throw for invalid item id', async () => {
+      const response = await app.inject({
+        method: HttpMethod.Delete,
+        url: `/items/invalid/tags/${v4()}`,
+      });
+      expect(response.statusCode).toBe(StatusCodes.BAD_REQUEST);
+    });
+    it('Throw for invalid tag id', async () => {
+      const response = await app.inject({
+        method: HttpMethod.Delete,
+        url: `/items/${v4()}/tags/invalid}`,
+      });
+      expect(response.statusCode).toBe(StatusCodes.BAD_REQUEST);
+    });
+
+    describe('Signed out', () => {
+      it('Cannot delete tag', async () => {
+        const response = await app.inject({
+          method: HttpMethod.Delete,
+          url: `/items/${v4()}/tags/${v4()}`,
+        });
+        expect(response.statusCode).toBe(StatusCodes.UNAUTHORIZED);
+      });
+    });
```

Comment:
How does this differ from the two previous tests ? I do not see explicitly that you deactivate login mocks. Maybe you forgot them on the other two tests ?

Reply:
Previous tests have incorrect body, and it throws because of that even before checking for an authenticated user. This test can go through the authentication check because the request is correct.